### PR TITLE
Config Validation, List Issues

### PR DESCRIPTION
**Pull request by SWW13**
- added config schema validation
  - small changes to config schema
- added issue list
  - added labels
  - removed updated timestamp

```
syso-ws17-grpXX: (forks: 2, last push: 2017-10-22 17:12:19)
    pr 1: HW0: Abgabe (created: 2017-10-22 16:21:31, labels: [], assignees: [])
        https://github.com/htwg-syslab-syso/syso-ws17-grpXX/pull/1
        state: open, merge status: clean
        + 24, - 1 (3 files changed)
    issue 6: Frage zu CONIG_* (created: 2017-10-28 08:10:49, labels: [question], assignees: [])
        https://github.com/htwg-syslab-syso/syso-ws17-grpXX/issues/6
```

### CODE REVIEW

File: config/syslab-reviewer.sample.yml [hunk 1]
```diff
@@ -2,13 +2,11 @@ github:
   token: your_token
 
 courses:
-  bsys:
+  -
```
Comment (steveeJ) [inline config/syslab-reviewer.sample.yml]: Is this single dash required to write a dictionary as a list item?

Comment (SWW13) [reply config/syslab-reviewer.sample.yml]: Yes, the dash starts a sequence item. In this case the item is a mapping.

Replacing the mapping with a sequence removes the duplicate name of the course and simplifies schema validation.

**Comment by SWW13** [comment]
Added missing schema file.

File: config/syslab-reviewer.sample.yml (hunk 1 above)
Comment (steveeJ) [reply config/syslab-reviewer.sample.yml]: It should still be valid if the mapping is started in the same line with the `-`, as in 
```yml
- name: BSYS
  organization: htwg-syslab-bsys
  repo_regex: bsys-(ss|ws)\d{2}-grp\d+
```

**Comment by SWW13** [comment]
removed unneeded newline in courses sequence

File: config/syslab-reviewer.schema.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      token:
+        type: str
+        required: true
+
+  courses:
+    name: List of courses
+    type: seq
+    required: true
+    sequence:
+    -
```
Comment (steveeJ) [inline config/syslab-reviewer.schema.yml]: unnecessary newline here

**Comment by SWW13** [comment]
removed unneeded newline in schema validation file

**Pull request merged**